jx_sqlite/expressions.py

```diff
@@ -28,7 +28,7 @@
 
 @extend(Variable)
 def to_sql(self, schema, not_null=False, boolean=False):
-    cols = [c for cname, cs in schema.map_to_sql().items() if startswith_field(cname, self.var) for c in cs]
+    cols = list({c for cname, cs in schema.map_to_sql(self.var).items() for c in cs})
```

Comment:
I suggest:

    [c for cname, cs in schema.map_to_sql(self.var).items() for c in cs]